components/modal/modalContent.css

```diff
@@ -0,0 +1,38 @@
+.ui-modal-content-block {
+  background-color: var(--tx-generic-color-blank);
+  border-radius: 8px;
+}
+
+.ui-modal-content-block__title {
+  font-weight: 600;
+  border-bottom: 2px solid var(--tx-modal-content-background-color);
+  padding: 24px;
+}
+
+.ui-modal-content-block__container {
+  border-radius: 8px;
```

Comment:
we have variable for `border-radius` - `--tx-generic-border-radius` (also should be fixed in other places)